Resources/Prototypes/_LostParadise/Maps/normandy.yml

```
@@ -77,0 +1,76 @@
+- type: gameMap
```

yamllint

[error] wrong new line character: expected \n

(new-lines)